api/app/models/common/areas_of_interest.py

```diff
@@ -62,32 +62,32 @@ def set_version_default(cls, v):
 
 class KeyBiodiversityAreaOfInterest(AreaOfInterest):
     type: Literal["key_biodiversity_area"] = "key_biodiversity_area"
-    id: str = Field(
-        ..., title="Key Biodiversity Area site code", examples=["36", "18", "8111"]
+    ids: List[str] = Field(
+        ..., title="Key Biodiversity Area site codes", examples=[["36"], ["18", "8111"]]
     )
 
 
 class ProtectedAreaOfInterest(AreaOfInterest):
     type: Literal["protected_area"] = "protected_area"
-    id: str = Field(
+    id: List[str] = Field(
```

Comment:
Shouldn't all of these AOI fields be renamed to `ids`?
```suggestion
    ids: List[str] = Field(
```